install.sh

```diff
@@ -3,3 +3,5 @@
 python setup.py install
 python manage.py migrate --settings=graphspace.settings.local
 bower install
```

Comment:
Comment its temporary until all library versions upgraded, currently we have to forcefully put the version